.github/workflows/main.yml

```diff
@@ -65,6 +69,143 @@ jobs:
         with: 
           name: linux 
           path: workspaces/**/Cargo.lock
+      - name: Prepare deployment artifacts
+        run: |
+          # Strip binaries
+          strip \
+          workspaces/host/target/debug/freestanding-execution-engine \
+          workspaces/host/target/debug/generate-policy \
+          workspaces/linux-host/target/debug/veracruz-client \
+          workspaces/linux-host/target/debug/veracruz-server \
+          workspaces/linux-runtime/target/debug/runtime_manager_enclave
+          # Copy artifacts to new directory
+          mkdir -p artifacts
+          cp -a \
+          proxy_cleanup.sh \
+          workspaces/ca-cert.conf \
+          workspaces/cert.conf \
+          workspaces/host/target/debug/freestanding-execution-engine \
+          workspaces/host/target/debug/generate-policy \
+          workspaces/linux-host/target/debug/veracruz-client \
+          workspaces/linux-host/target/debug/veracruz-server \
+          workspaces/linux-runtime/target/debug/runtime_manager_enclave \
+          artifacts/
+      - name: Upload deployment artifacts
+        id: upload-deployment-artifacts
+        uses: actions/upload-artifact@v3
+        with: 
+          name: linux_deployment_artifacts
+          path: |
+            artifacts/proxy_cleanup.sh
+            artifacts/ca-cert.conf
+            artifacts/cert.conf
+            artifacts/freestanding-execution-engine
+            artifacts/generate-policy
+            artifacts/veracruz-client
+            artifacts/veracruz-server
+            artifacts/runtime_manager_enclave
+
+  vod-full-deployment:
+    runs-on: ubuntu-latest
+    needs: [linux]
+    container:
+      image: ghcr.io/veracruz-project/veracruz/ci@sha256:38add618b42f59a89fdd4c46831dcf9c9e73b159bc74c87251c8263a2ec6238a
+      volumes:
+        - ${{ github.workspace }}:/work/video-object-detection
+    steps:
+      - name: Check out the VOD repository
+        uses: actions/checkout@v3
+        with:
+          repository: 'veracruz-project/video-object-detection'
+          ref: 'main'
+          submodules: recursive
+          set-safe-directory: true
+      - name: Build
+        run: |
+          # grab every bash code block for this step, remove line continuation,
```

Comment:
This 9-line pattern is now repeated 6 times in the file. I wouldn't insist on it, but it might we worth moving it into one place, which would presumably have to be a new file, unfortunately, so it's not a big win.

Reply:
Let's refactor this in another PR